react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -192,26 +189,14 @@ class ParticipantView extends Component<Props> {
             onPress,
             _avatar: avatar,
             _connectionStatus: connectionStatus,
+            _renderVideo,
```

Comment:
con we rename it to `renderVideo` here, like we do with `avatar` and so on? That will make the diff even smaller.